crypto/evp/evp_enc.c

```diff
@@ -1441,9 +1441,10 @@ static void set_legacy_nid(const char *name, void *vlegacy_nid)
 #endif
 
 static void *evp_cipher_from_dispatch(const int name_id,
-                                      const OSSL_DISPATCH *fns,
+                                      const OSSL_ALGORITHM *algodef,
```

Comment:
should the function be renamed to evp_cipher_from_algorithm? And similarly for other evp_xxx_from_dispatch?

Reply:
Sure...